src/mpid/ch4/generic/mpidig_recv.h

```diff
@@ -54,6 +54,8 @@ static inline int MPIDI_handle_unexpected(void *buf,
     MPIR_FUNC_VERBOSE_STATE_DECL(MPID_STATE_MPIDI_HANDLE_UNEXPECTED);
     MPIR_FUNC_VERBOSE_ENTER(MPID_STATE_MPIDI_HANDLE_UNEXPECTED);
 
+    /* Calculate the size of the data from the active message information and update the request
+     * object. */
     in_data_sz = MPIDI_CH4U_REQUEST(rreq, count);
```

Comment:
Should we put ch4 comment enhancement in this commit ?

Reply:
This could potentially move over to the prereq branch. I had a patch that added a bunch of comments, but it also included some non-posix stuff.

Reply:
I've moved this change to the prereq branch. It still shows up here, but as a separate patch.